app/code/Magento/Customer/Ui/Component/Listing/AttributeRepository.php

```diff
@@ -143,9 +143,13 @@ protected function getOptionArray(array $options)
     {
         /** @var \Magento\Customer\Api\Data\OptionInterface $option */
         foreach ($options as &$option) {
+            $value = $option->getValue();
+            if ($option->getOptions()) {
```

Comment:
Since the `OptionsInterface` does not have strict return types on its functions, the `getOptions` may not return an array, which may lead to a problem since the `getOptionArray` does require an array.

```suggestion
            if (is_array($option->getOptions())) {
```

What do you think?